haystack/nodes/prompt/prompt_node.py

```diff
@@ -267,7 +271,7 @@ def __init__(
         For more details about these kwargs, see
         Hugging Face [documentation](https://huggingface.co/docs/transformers/en/main_classes/pipelines#transformers.pipeline).
         """
-        super().__init__(model_name_or_path, max_length)
```

Comment:
I removed `max_length` here because `HFLocalInvocationLayer` requires the model to be loaded to be able to set a default `max_length` value if the user passes `max_length=None` to the `PromptNode`.